opencog/cython/opencog/Cast.cc

```diff
@@ -8,3 +8,4 @@ using namespace opencog;
 Handle atom_from_the_void(long p) { return *((Handle*) p); }
 long void_from_candle(const Handle& hp) { return (long) (&hp); }
 long void_from_cptr(Handle* hp) { return (long) (hp); }
+long avoid_from_cptr(Handle* hp) { return (long) (hp->operator->()); }
```

Comment:
(but that comment is really about the value of UUIDs wrapped in Handles, not about Handles as wrappers for pointers)